tests/test_units/test_contents.py

```diff
@@ -86,7 +86,7 @@ def test_content_fluid_contact_raises_on_invalid_contact(regsurf, globalconfig2)
         ExportData(
             config=globalconfig2,
             name="MyName",
-            content={"fluid_contact": {"contact": "OEC"}},
+            content={"fluid_contact": {"contact": "oec"}},
```

Comment:
changed this to lowercase to avoid the UserWarning on top of the ValueError that the test is mainly checking.

Reply:
Now that you mention it.. I think we should, it is as you say a very common convention (more common than lowercase letters IMO). Then we can convert it to lowercase for them without emitting the message 😊

I will remove the change here then 🙂